### DISCUSSION TITLE
v0.45.1 version number confusion

### DISCUSSION THREAD

**Discussion by Ororaa** [nomi-san/parsec-vdd#31] [Q&A] [answered]
Hello
I have reinstalled from the v0.45.1 release page but the app is still saying v0.45.0. Is this the application version or is this referring to the API version listed in the readme? 
![image](https://github.com/nomi-san/parsec-vdd/assets/99012438/1bb55470-4431-4fdc-83fa-6c56a3f4d2ab)

Also is there a high-level feature changelog anywhere? I am not sure what has changed with this update - I am looking forward to having it load displays on start-up but I'm not sure if it's implemented in this version yet!

Thank you for your work on this and for any help!

**Comment by nomi-san** [comment, marked as answer]
It's just a tag renaming from v0.45-a -> v0.45.1.
The next patch 0.45.2 will be published soon, including this feature.

**Reply by Ororaa** [reply]
Okay great, thank you!
